src/components/post-conditions.tsx

```diff
@@ -166,6 +176,7 @@ const Condition = ({ condition, isLast }: { condition: PostCondition; isLast?: b
             ? ftDecimals((condition as any).amount, ftMetadata?.decimals || 0)
             : getAmount(condition)}{' '}
           {ftMetadata?.symbol || getConditionTicker(condition)}
+          {condition.type === 'stx' && <StxPriceButton tx={tx} value={Number(tx.fee_rate)} />}
```

Comment:
It looks like this is currently displaying the tx fee, but we actually need to display the dollar amount of the STX value instead. In this example, it would be the dollar value for 2 STX.

Reply:
@He1DAr Thanks for mentioning the correction. I wrote a function to get the dollar value of stx. 
Kindly review it. Thanks again!